src/core/icons/folderIcons.ts

```diff
@@ -781,7 +781,7 @@ export const folderIcons: FolderTheme[] = [
       { name: 'folder-windows', folderNames: ['windows', 'win', 'win32'] },
       {
         name: 'folder-macos',
-        folderNames: ['macos', 'mac', 'osx', 'DS_Store'],
+        folderNames: ['macos', 'mac', 'osx', 'DS_Store', 'iPhone', 'iPad', 'iPod', 'macbook', 'macbook-air', 'macosx', 'apple'],
```

Comment:
You basically need to format the code the as the linting error report says:

```suggestion
        folderNames: [
          'macos',
          'mac',
          'osx',
          'DS_Store',
          'iPhone',
          'iPad',
          'iPod',
          'macbook',
          'macbook-air',
          'macosx',
          'apple'
        ],
```

You can do it manually (editing the code), or you can run the `bun run format` commmand in your terminal (you need [`bun` installed in your computer](https://bun.sh/)) and it will do it automatically